src/src/lms/lms.controller.ts

```diff
@@ -0,0 +1,69 @@
+import {
+	Body,
+	Controller,
+	Delete,
+	Get,
+	Param,
+	Patch,
+	Post,
+	Req,
+	Res,
+	UseGuards,
+	UseInterceptors,
+	UsePipes,
+	ValidationPipe,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+import { SentryInterceptor } from 'src/common/interceptors/sentry.interceptor';
+import { AuthGuard } from 'src/modules/auth/auth.guard';
+import { SearchLMSDto } from './dto/search-lms.dto';
+import { LMSService } from './lms.service';
+import { LSMTestTrackingDto } from './dto/lms-test-tracking.dto';
+
+@UseInterceptors(SentryInterceptor)
+@Controller('lms')
+export class LMSController {
+	constructor(private readonly lmsService: LMSService) {}
+
+	//test CRUD
+
+	//user test allow
+	@Get('/testTracking')
+	@UseGuards(new AuthGuard())
+	getTestAllowStatus(@Req() header: Request, @Res() response: Response) {
```

Comment:
@snehal0904 @rushi-tekdi is it getTestAllowStatus or getTestTrackingStatus? 
For which all tests?

Reply:
this is not future proof, add another task in tracker for same for TODO